style.css

```diff
@@ -0,0 +1,247 @@
+*{
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+input, button {
+  font-family: inherit;
+}
+
+body {
+  background: url(img/Background.png) no-repeat;
+  background-size: cover;
+  font-family: 'Roboto', sans-serif;
+  margin: 0 auto;
+  min-width: 320px; 
+  max-width: 1440px;
+  /*min-height: 1024px;*/
+}
+
+.main {
+  max-width: 90%;
+  margin: auto;
+}
+
+.container,
+.three/*,
+.six,
+.ten */{
+  min-height: 1024px;
+}
+
+.level {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 4.8%;
+}
+
+.level__choice {
+  font: normal bold 57px/ 67px 'Roboto';
+  color: #FFFFFF;
```

Comment:
Codacy found an issue: [Expected "#FFFFFF" to be "#ffffff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5838315)